odysseus/convertFtbQuests.ts

```diff
@@ -494,23 +531,29 @@ function convertTask(task: QuestTask, questFile: QuestFile): HeraclesQuestTask {
                 if (task.item.id === 'itemfilters:tag' && task.item.tag?.value) {
                     return {
                         type: 'heracles:item',
-                        amount: task.count,
+                        title: task.title,
+                        icon: task.icon ? convertIcon(task.icon) : undefined,
+                        amount: task.count ? parseInt(task.count) : undefined,
                         item: `#${task.item.tag?.value as ResourceLocation}`,
                         collection_type: collectionType
                     }
                 }
 
                 return {
                     type: 'heracles:item',
-                    amount: task.count,
-                    item: task.item.id,
+                    title: task.title,
+                    icon: task.icon ? convertIcon(task.icon) : undefined,
+                    amount: task.count ? parseInt(task.count) : undefined,
+                    item: task.item.id == 'ftbquests:book' ? 'heracles:quest_book' : task.item.id,
```

Comment:
This check is used before, should probably have an item sanitation function.

Reply:
the book conversion is used in 3 spots, if that's what you mean? they have different types for their items though

Reply:
That's fine, just something like `getItemId(id: ResourceLocation)`

Reply:
added in 4e093f9